src/components/Public/Page/Attachment/ListLoader.vue

```diff
@@ -244,8 +252,11 @@ export default {
         const namespaceID = this.namespace.namespaceID
         for (const [index, attachmentID] of Object.entries(set)) {
           if (typeof attachmentID === 'string') {
+            this.processing = true
```

Comment:
We'll need to do the Promise.all here aswell. Promise.all(Object.entries(set).map(([index, attachmentID]) => {}).finally

Make sure to return the 256 promise instead of awaiting it